README.md

```diff
@@ -189,6 +189,8 @@ A collection of resources to learn object-oriented programming and related conce
 * [The MVC Pattern and PHP, Part 1](http://www.sitepoint.com/the-mvc-pattern-and-php-1/) and [The MVC Pattern and PHP, Part 2](http://www.sitepoint.com/the-mvc-pattern-and-php-2/) - An introduction tutorial about MVC pattern.
 * [Alternatives To MVC](http://blog.ircmaxell.com/2014/11/alternatives-to-mvc.html) - An introduction to alternative architectures to MVC.
 * [N-Tier Architecture - An Introduction](http://blog.ircmaxell.com/2012/08/n-tier-architecture-introduction.html) - An introduction to n-tier architecture.
+* [Hexagonal Architecture](http://fideloper.com/hexagonal-architecture) - An introduction to also called Ports and Adapters architecture by Chris Fidao
```

Comment:
The language is a little off at "An introduction to also called". Did you accidentally delete part of your sentence?